src/main/java/org/geppetto/core/simulation/ISimulatorCallbackListener.java

```diff
@@ -55,4 +55,5 @@
 	 */
 	void endOfSteps(IAspectConfiguration aspectConfiguration, Map<File, ResultsFormat> results) throws GeppettoExecutionException;
 
+	void externalProcessFailed(String message, Exception e);
```

Comment:
@slarson This was added to get an error message from the external processes to the experiment run thread class in the simulation bundle. The Experiment Run Thread implements this interface, and whenever something goes wrong in the external processes (which run in own thread) it receives a message through this method.